inference-engine/tests/functional/plugin/shared/include/subgraph_tests/const_strided_slice_concat.hpp

```diff
@@ -0,0 +1,17 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "shared_test_classes/subgraph/const_strided_slice_concat.hpp"
+
+namespace SubgraphTestsDefinitions {
+
+TEST_P(ConstStridedSliceConcatTest, CompareWithRefImpl) {
+    LoadNetwork();
```

Comment:
Why don't you just call Run()?

Reply:
Good point. I will simplify with Run(). Thanks